### PR TITLE
Version drift fixes

### PR DESCRIPTION
**Pull request by OR13**
kid was being leaked into test vectors, its been removed.

### CODE REVIEW

**Review by gjgd — COMMENTED** [review]
LGTM once tests are passing

File: packages/test-vectors/src/sidetree-universal-wallet/wallet-keypair.json [hunk 1]
```diff
@@ -9,15 +9,13 @@
         "publicKeyJwk": {
           "crv": "Ed25519",
           "x": "BnamLr9mV2goDx4gfFzaBCBXWy2Bpa64cNNhVMIe4vc",
-          "kty": "OKP",
-          "kid": "DrUSp0MnN8txc0J-odEXDRu-QRRDpOP9c2awKGW2Ozk"
+          "kty": "OKP"
```
Comment (gjgd) [inline packages/test-vectors/src/sidetree-universal-wallet/wallet-keypair.json:12]: Also need to update the code that generated those fixtures, in `core`

**Review by gjgd — CHANGES_REQUESTED** [review]

File: packages/test-vectors/src/sidetree-universal-wallet/wallet-keypair.json (hunk 1 above)
Comment (OR13) [reply packages/test-vectors/src/sidetree-universal-wallet/wallet-keypair.json:12]: these fixtures are generated by the wallet package, not core. I will check core, but should be seeing test failures if the was fixture drift.

Comment (OR13) [reply packages/test-vectors/src/sidetree-universal-wallet/wallet-keypair.json:12]: tested locally, there are failures, I will update the fixtures.

Comment (gjgd) [reply packages/test-vectors/src/sidetree-universal-wallet/wallet-keypair.json:12]: Looks like core tests are failing https://github.com/transmute-industries/sidetree.js/runs/1298261123#step:9:248

Comment (OR13) [reply packages/test-vectors/src/sidetree-universal-wallet/wallet-keypair.json:12]: yep, thats a good sign, ci is catching fixture drift.

**Comment by OR13** [comment]
`@sidetree/ethereum: Segmentation fault (core dumped)` 
r-u-tx4igl

**Review by gjgd — APPROVED** [review]
LGTM

**Pull request merged**